src/index.css

```
@@ -396,4 +396,25 @@ input::placeholder {
+  }
+}
+
+.skeleton-shimmer {
+  background: linear-gradient(
+    90deg,
+    var(--color-surface-mid) 25%,
+    var(--color-surface-highest) 50%,
+    var(--color-surface-mid) 75%
+  );
+
+  background-size: 200% 100%;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)